### PR TITLE
Fix CI by reverting to coq_makefile

### LINKED ISSUE #75: make install fails
The manual install as specified in the README does not work:
```
$ make install
make: *** No rule to make target 'install'.  Stop.
```
`dune` also does nothing unless one sets some parameters (?, man page of dune and dune documentation say nothing about `mandir`)
```
$ dune install
Error: The mandir installation directory is unknown.
Hint: It could be specified with --mandir
```

I have set my installation path via `$COQLIBINSTALL=/home/user/.local/share/coq`.

### PR DESCRIPTION
**Pull request by pi8027**
Let me run CI here since the multinomials package does not seem to compile with coq-master.

Closes #75
Closes #76

### CODE REVIEW

**Comment by pi8027** [comment]
Does anyone know how to fix this error? @gares @proux01
```
  # File "./src/xfinmap.v", line 2, characters 0-68:
  # Error: Cannot load elpi_elpi.dummy: no physical path bound to elpi_elpi
```

**Comment by proux01** [comment]
Unforunately, I fear this is a dune related issue and I have no clue what dune is doing wrong.
The easiest and fastest fix is probably to revert to coq_makefile here for now.

File: src/dune [hunk 1]
```diff
@@ -8,6 +8,7 @@
   -w -projection-no-head-constant)
  (theories
   elpi
+  elpi_elpi
```
Comment (pi8027) [inline src/dune:11]: The source of the issue seems to be the fact that Coq-Elpi uses two prefixes of logical paths `elpi` and `elpi_elpi`. So this line fixes the issue. However, I see this FIXME comment on the Coq-Elpi side: https://github.com/LPCIC/coq-elpi/blob/c13ecc9182c4344c00894d0523ccba1d5c03e6d8/elpi/dune#L2

@gares Can you confirm that this is the right fix? Or is it reasonable to fix the issue on the Coq-Elpi side?

Also CC Dune experts: @ejgallego @Alizter

**Comment by pi8027** [comment]
Also, this patch breaks the build of multinomials with Coq <= 8.18...

**Comment by Alizter** [comment]
Does the version of elpi change between Coq versions? It seems strange that the coq.theory elpi_elpi is missing only for older versions?

**Comment by Alizter** [comment]
Looks like the version of `coq-elpi` is changing between the mathcomp versions. In `2.2.0` is requires the extra `elpi_elpi` theory that was added. I don't really have time to help hacking around this and `dune` doesn't really support the way elpi requires things to be built (nor do we have the manpower to improve this at the moment). So perhaps the best thing to do is revert to coq_makefile here.

**Comment by pi8027** [comment]
@proux01 @Alizter Thanks for the feedback.

Since I couldn't find a way to make the build of multinomials compatible both with Coq 8.18 and master, I'm going to revert it to coq_makefile now. CC: @strub.

**Comment by pi8027** [comment]
This PR should also close #75 and #76. @haansn08

**Comment by pi8027** [comment]
Since this PR is blocking the merge of coq-community/apery#24, I'm merging this now, but please let me know if you see any issues.

**Pull request merged**